test/e2e-common.sh

```diff
@@ -289,7 +294,7 @@ function install_channel_crds() {
   ko apply --strict -f ${TMP_IN_MEMORY_CHANNEL_CONFIG_DIR} || return 1
 
   # TODO(https://github.com/knative/eventing/issues/3590): Enable once IMC chaos issues are fixed.
-  # scale_controlplane imc-controller imc-dispatcher
```

Comment:
Could we split this into imc-controller / imc-dispatcher so we can debug control plane events from the dataplane events. I'm seeing (and totes not surprised) that many tests are failing with the dataplane going bad. This way we can focus one bit at a time?